practice-one/src/Components/Section/Footer.jsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const FooterLogo = ({ logoSrc, altText }) => (
+  <div className="card-header">
+    <a href="/" className="logo">
+      <img src={logoSrc} alt={altText} />
+    </a>
+  </div>
+);
+
+FooterLogo.propTypes = {
+  logoSrc: PropTypes.string.isRequired,
+  altText: PropTypes.string.isRequired,
+};
+
+const FooterCard = ({ children }) => <div className="card w-[30%] lg:w-[20%]">{children}</div>;
+
+FooterCard.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const FooterCardBody = ({ text }) => (
+  <div className="card-body pt-5 pb-5">
+    <p className="text-sm leading-medium">{text}</p>
+  </div>
+);
+
+FooterCardBody.propTypes = {
+  text: PropTypes.string.isRequired,
+};
+
+const FooterMenu = ({ title, children }) => (
+  <div className="footer-menu nav-menu">
+    <ul className="menu-list text-sm">
+      <a href="/" className="item font-bold hover:text-primary">
+        {title}
+      </a>
+      {children}
+    </ul>
+  </div>
+);
+
+FooterMenu.propTypes = {
+  title: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+};
+
+const FooterMenuItem = ({ text }) => (
+  <li className="list-item pt-4">
+    <a className="item hover:text-secondary focus:tex-pri" href="/">
+      {text}
+    </a>
+  </li>
+);
+
+FooterMenuItem.propTypes = {
+  text: PropTypes.string.isRequired,
+};
+
+const Footer = () => (
+  <footer className="footer bg-white relative mt-56 lg:mt-0 m-7">
+    <div className="footer-content container mx-auto mt-14 mb-12 flex flex-col lg:flex-row flex-wrap justify-between gap-y-8 lg:flex-nowrap">
+      <FooterCard>
+        <FooterLogo logoSrc="src\assets\logo.png" altText="logo" />
+        <FooterCardBody text="We built an elegant solution. Our team created a fully integrated sales and marketing solution for SMBs" />
+        <div className="card-footer">
+          <ul className="media menu-list flex gap-5">
+            <li>
+              <a className="item" href="https://twitter.com/">
+                <img
+                  src="src\assets\Twitter.png"
+                  alt="Here is a logo of the media above"
+                />
+              </a>
+            </li>
+            <li>
+              <a className="item" href="https://www.facebook.com/">
+                <img
+                  src="src\assets\Facebook.png"
+                  alt="Here is a logo of the media above"
+                />
+              </a>
+            </li>
+            <li>
+              <a className="item" href="https://www.linkedin.com/">
+                <img
+                  src="src\assets\Linkedin.png"
+                  alt="Here is a logo of the media above"
+                />
+              </a>
+            </li>
+          </ul>
+        </div>
+      </FooterCard>
+      <FooterMenu title="Company">

```

Comment:
why don't use `map`. Please keep in mind that if it's react and we have a list, we should use `map` to render into the UI. Do not repeat this chunk code

Reply:
Yes,I missed them, so I updated the lists and used map to render into UI in the latest commit